xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -277,6 +277,8 @@ extern "C" {
   const unsigned int PVR_TIMER_TYPE_REQUIRES_EPG_SERIESLINK_ON_CREATE = 0x02000000; /*!< @brief this type should not appear on any create menus which don't provide an associated EPG tag with a series link */
   const unsigned int PVR_TIMER_TYPE_SUPPORTS_READONLY_DELETE          = 0x04000000; /*!< @brief this type allows deletion of an otherwise read-only timer */
   const unsigned int PVR_TIMER_TYPE_IS_REMINDER                       = 0x08000000; /*!< @brief timers of this type do trigger a reminder if time is up by calling the Kodi callback 'ReminderNotification'. */
+  const unsigned int PVR_TIMER_TYPE_SUPPORTS_START_MARGIN             = 0x10000000; /*!< @brief this type supports pre record time (PVR_TIMER.iMarginStart) */
+  const unsigned int PVR_TIMER_TYPE_SUPPORTS_END_MARGIN               = 0x20001000; /*!< @brief this type supports post record time (PVR_TIMER.iMarginEnd) */
```

Comment:
Was here a typo (the 0x1000) or to have also `PVR_TIMER_TYPE_SUPPORTS_START_END_MARGIN` inside defined?

If correct it becomes on rework:
```cpp
  typedef enum PVR_TIMER_TYPES
  {
    ...
    PVR_TIMER_TYPE_SUPPORTS_READONLY_DELETE = (1 << 26),
    PVR_TIMER_TYPE_IS_REMINDER = (1 << 27),
    PVR_TIMER_TYPE_SUPPORTS_START_MARGIN = (1 << 28),
    PVR_TIMER_TYPE_SUPPORTS_END_MARGIN = (1 << 29) | PVR_TIMER_TYPE_SUPPORTS_START_END_MARGIN,
  } PVR_TIMER_TYPES;
```

Reply:
Oooops, this is a typo. Will fix.